tests/test_sims.py

```diff
@@ -882,3 +882,142 @@ def test_kwarg_passing(self):
         g = rdpg(X, rescale=True, loops=False, directed=False)
         self.assertTrue(is_symmetric(g))
         self.assertTrue(is_loopless(g))
+
+
+class Test_MMSBM(unittest.TestCase):
```

Comment:
I don't see any tests asserting that the graph you sample is anywhere near some expected answer. I realize it is harder in this case cause of the loose notion of membership, but you should be able to at least think of something simple/not super foolproof

Reply:
I could generate an ER by assigning all nodes to a single community and check that the probability obtained is the same as that indicated by the B matrix such as I do on the labels. 

Or I could produce and sbm and check that the latent positions to an sbm generated from same block matrix are close enough. Would that be okay?